templates/interface/filter-panels.interface.hbs

```diff
@@ -58,9 +58,26 @@
         </div>
       </div>
 
+      <div class="form-group clearfix" id="filter-value-type">
+        <div class="col-sm-4 control-label">
+          <label>Value type</label>
+        </div>
+        <div class="col-sm-8">
+          <label for="value-type-field-{{id}}" class="select-proxy-display">
+            <select name="value-type-field-{{id}}" id="value-type-field-{{id}}" data-field="valueType" class="hidden-select form-control">
+              <option value="enter-value">Enter a value</option>
```

Comment:
I prefer the HTML template contains only `<option value="enter-value">Enter a value</option>` for now. This way each PR represents a fully functional feature without broken features. The current PR contains 3 useless dropdown options that don't do what it says. All the JavaScript are fine.

@squallstar Do you agree with the principle here? ☝️

Reply:
Agreed.

Reply:
@squallstar , @tonytlwu
https://screencast-o-matic.com/watch/cYlQIqNMzg
removed unused options, i will add them in next tickets

Reply:
I can't see any new commit here, did you push your changes?

Reply:
@squallstar 
Strange, the console showed that I uploaded all the latest changes, but the commists did not really appear